mods/example/mod.chrome.yaml

```diff
@@ -54,6 +53,7 @@ ChromeLayout:
 	common|chrome/settings-input.yaml
 	common|chrome/text-notifications.yaml
 	common|chrome/tooltips.yaml
+	example|chrome/ingame-observer.yaml
```

Comment:
Suggest you move it up to the other ingame yamls.

Reply:
Done.